MutexLocks/MutexObject.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace MutexLocks
+{
+    public class MutexObject : IDisposable
```

Comment:
This class feels rather pointless to me. All you've done is decorated an IDisposable with another class that adds an alias to the one method that IDisposables already have. I was expecting to see some mutex locking behavior in here.